templates/citizen_project/show.html.twig

```diff
@@ -1,31 +1,41 @@
 {% extends 'citizen_project/show_base.html.twig' %}
 
-{% block citizen_project_show_content %}
+{% block javascripts %}
     {{ parent() }}
-    <div class="citizen-project-next-actions">
-        {% for action in citizen_actions %}
-            <ul class="text--body l__row l__row--h-stretch b__nudge--bottom">
-                <li class="text--orange text--bold text--center text--medium-small b__nudge--right">
-                    {{ action.beginAt|date('d') }}
-                    <br/>
-                    {{ action.beginAt|localizeddate('full', 'none', format="MMM")|capitalize }}
-                </li>
-
-                <li class="b__flex--1">
-                    <a href="{{ path('app_citizen_action_show', {slug: action.slug}) }}"
-                       class="text--bold text--medium-small link--no-decor">{{ action.name }}</a>
-                    <div class="text--small text--gray">{{ action.participantsCount }} inscrit(s)</div>
-                </li>
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+    <script type="text/javascript">
+        Kernel.onLoad(function(di) {
+            $('#show_more_citizen_actions').click(function(){
+                $('#show_more_citizen_actions').toggleClass('visually-hidden');
+                $('#show_less_citizen_actions').toggleClass('visually-hidden');
+                $('#other_citizen_actions').toggleClass('visually-hidden');
+            });
+            $('#show_less_citizen_actions').click(function(){
```

Comment:
IMO you can use just one function with a class, these two functions are identical.